From 42bc999b952584d170694424754a643000dc7c21 Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Sun, 20 Jul 2014 11:43:49 +0200
Subject: [PATCH] cmake: remove installation of basket.desktop using
 xdg-desktop-menu

basket.desktop is already installed correctly from src/CMakeLists.txt,
so no need to do it again (and wrongly) here.
---
 file-integration/CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

Forwarded: https://github.com/gl-bars/basket/pull/14
diff --git a/file-integration/CMakeLists.txt b/file-integration/CMakeLists.txt
index 3f8013e..ac7e362 100644
--- a/file-integration/CMakeLists.txt
+++ b/file-integration/CMakeLists.txt
@@ -10,14 +10,9 @@ install(TARGETS basketthumbcreator DESTINATION ${PLUGIN_INSTALL_DIR})
 ########### install files ###############
 
 find_package(SharedMimeInfo REQUIRED)
-find_program(XDG-DESKTOP-MENU_EXECUTABLE xdg-desktop-menu)
 
 install(FILES  basketthumbcreator.desktop DESTINATION ${SERVICES_INSTALL_DIR})
 install(FILES basket.xml DESTINATION ${XDG_MIME_INSTALL_DIR})
 update_xdg_mimetypes(${XDG_MIME_INSTALL_DIR})
-install(CODE "
-execute_process(COMMAND ${XDG-DESKTOP-MENU_EXECUTABLE} install
-  --novendor ${CMAKE_SOURCE_DIR}/src/basket.desktop)
-")
 
 kde4_install_icons(${ICON_INSTALL_DIR})
-- 
2.0.1

